src/components/Layout/LayoutWrapper.test.jsx

```diff
@@ -41,4 +42,14 @@ describe('LayoutWrapper styles', () => {
     expect(firstChild).toHaveStyle('width: 100%');
     expect(firstChild).toHaveStyle('height: 100%');
   });
+
+  it('applies background for light theme', async () => {
+    const { firstChild } = (await build()).container;
+    expect(firstChild).toHaveStyle(`background: ${lightTheme.color.background}`);
+  });
+
+  it('applies background for dark theme', async () => {
+    const { firstChild } = (await build(<LayoutWrapper/>, darkTheme)).container;
+    expect(firstChild).toHaveStyle(`background: ${darkTheme.color.background}`);
+  });
```

Comment:
These are good test cases 👏🏻

Reply:
OMG, I wasn't expecting a review so soon.
I forgot to add a screenshot of the test coverage, I also added an update.
This is the new coverage:
![image](https://user-images.githubusercontent.com/78768286/111707772-498d9c80-880a-11eb-87ed-2ea0906230c1.png)
Given that the `gapi` object is available once the `youtube` script is loaded, I wasn't being checking that behavior.